test/runtime/ct.go

```diff
@@ -0,0 +1,433 @@
+		case HTTPDummy:
+			cmd := fmt.Sprintf(
+				"curl -s --fail -o /dev/null -w %%{http_code} --connect-timeout 5 http://%s/dummy",
+				net.JoinHostPort(dest, "80"))
+			res := vm.ContainerExec(src, cmd)
+			valid := false
+			if res.SingleOut() == "404" {
+				valid = true
+			}
+			ExpectWithOffset(1, valid).Should(assertFn(), "Failed to curl '%s'", res.GetCmd())
+		case HTTPPrivate:
+			res := vm.ContainerExec(src, helpers.CurlFail(fmt.Sprintf(
+				"http://%s/private", net.JoinHostPort(dest, "80"))))
+			ExpectWithOffset(1, res.WasSuccessful()).Should(assertFn(), "Failed to curl '%s'", res.GetCmd())
+		case HTTPPublic:
+			res := vm.ContainerExec(src, helpers.CurlFail(fmt.Sprintf(
+				"http://%s/public", net.JoinHostPort(dest, "80"))))
+			ExpectWithOffset(1, res.WasSuccessful()).Should(assertFn(), "Failed to curl '%s'", res.GetCmd())
+		case netcatPrivate:
+			cmd := fmt.Sprintf(`bash -c "python ./nc.py 1111 5 %s 80 "/private" | head -n 1 | grep \"HTTP/1.*200 OK\""`, dest)
+			res := vm.ContainerExec(src, cmd)
+			ExpectWithOffset(1, res.WasSuccessful()).Should(assertFn(), "Failed to httpRequest '%s'", res.GetCmd())
+		case netcatPublic:
+			cmd := fmt.Sprintf(`bash -c "python ./nc.py 1111 5 %s 80 "/public" | head -n 1 | grep \"HTTP/1.*200 OK\""`, dest)
+			res := vm.ContainerExec(src, cmd)
+			ExpectWithOffset(1, res.WasSuccessful()).Should(assertFn(), "Failed to httpRequest '%s'", res.GetCmd())
+		default:
+			Expect(true).To(BeFalse(), "Mode %s is not defined", mode)
+		}
+	}
+
+	BeforeEach(func() {
+		initialize()
+	})
+
+	It("tests conntrack tables between client to server", func() {
+		vm.SetPolicyEnforcement(helpers.PolicyEnforcementAlways)
+		containers(helpers.Create)
+		defer containers(helpers.Delete)
+		vm.WaitEndpointsReady()
+
+		meta := containersMeta()
+		identities, err := vm.GetEndpointsIdentityIds()
+		Expect(err).To(BeNil())
+
+		_, err = vm.PolicyImport(vm.GetFullPath(ctL4Policy), 300)
+		Expect(err).To(BeNil())
+
+		testCombinations := []connTest{
+			{meta[client], meta[server], helpers.IPv6, http},
+			{meta[client], meta[server], helpers.IPv4, http},
+			{meta[client2], meta[server], helpers.IPv6, http},
+			{meta[client2], meta[server], helpers.IPv4, http},
+			{meta[client], meta[server2], helpers.IPv6, http},
+			{meta[client], meta[server2], helpers.IPv4, http},
+			{meta[client2], meta[server2], helpers.IPv6, http},
+			{meta[client2], meta[server2], helpers.IPv4, http},
+		}
+
+		for _, testCase := range testCombinations {
+			testReach(testCase.src["Name"], testCase.destination[testCase.kind], testCase.mode, BeTrue)
+		}
+
+		beforeCT, err := vm.ExecCilium("bpf ct list global | wc -l").IntOutput()
+		Expect(err).To(BeNil())
+
+		beforeCTResults := map[string]int{}
+		for _, testCase := range testCombinations {
+			key := testCase.String()
+			data, err := countCTEntriesof(
+				testCase.destination[testCase.kind],
+				testCase.src[testCase.kind],
+				identities[testCase.src["Name"]])
```

Comment:
Make `"Name"` into a constant and utilize it throughout this test instead of hardcoding it.